x-pack/legacy/plugins/siem/public/components/open_timeline/edit_timeline_batch_actions.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiContextMenuPanel, EuiContextMenuItem, EuiBasicTable } from '@elastic/eui';
+import React, { useCallback, Dispatch } from 'react';
+import { TimelineDownloader } from './export_timeline/export_timeline';
+import { DeleteTimelineModalOverlay } from './delete_timeline_modal';
+import * as i18n from './translations';
+import { DeleteTimeline, DeleteTimelines, OpenTimelineResult } from './types';
+import { ExportTimeline } from './export_timeline';
+import { useEditTimelineActions } from './edit_timeline_actions';
+import { ActionToaster } from '../toasters';
+import { exportSelectedTimeline } from '../../containers/timeline/all/api';
+
+export const useEditTimelinBatchActions = ({
+  deleteTimelines,
+  dispatchToaster,
+  selectedItems,
+  tableRef,
+}: {
+  deleteTimelines: DeleteTimelines | undefined;
+  dispatchToaster: Dispatch<ActionToaster>;
+  selectedItems: OpenTimelineResult[] | undefined;
+  tableRef: React.MutableRefObject<EuiBasicTable<OpenTimelineResult> | undefined>;
+}) => {
+  const {
+    enableExportTimelineDownloader,
+    disableExportTimelineDownloader,
+    exportedIds,
+    isEnableDownloader,
+    isDeleteTimelineModalOpen,
+    onOpenDeleteTimelineModal,
+    onCloseDeleteTimelineModal,
+  }: DeleteTimeline & ExportTimeline = useEditTimelineActions(selectedItems);
+  const onCompleteBatchActions = useCallback(
+    (closePopover?: () => void) => {
+      if (closePopover != null) closePopover();
+      if (tableRef != null && tableRef.current != null) {
+        tableRef.current.changeSelection([]);
+      }
+      disableExportTimelineDownloader();
+      onCloseDeleteTimelineModal();
+    },
+    [tableRef.current]
```

Comment:
you are missing dependencies `disableExportTimelineDownloader` and `onCloseDeleteTimelineModal`